commit 786afa220e786b8967d41ae66082f122c040ceb7
Author: nat <nat@nekopon.pl>
Date:   Mon Aug 25 18:37:03 2025 +0200

    Fix for Python 3.13

diff --git a/src/utils.py b/src/utils.py
index a2e7028..ee03574 100644
--- a/src/utils.py
+++ b/src/utils.py
@@ -90,7 +90,14 @@ class extend_class(object):  # nocov
             )
 
         newAttributes = dict(decorated_cls.__dict__)
-        crossDelete = ["__abstractmethods__", "__module__", "_abc_impl", "__doc__"]
+        crossDelete = [
+          "_abc_impl",
+          "__abstractmethods__",
+          "__doc__",
+          "__firstlineno__",
+          "__module__",
+          "__static_attributes__"
+        ]
         [
             (newAttributes.pop(cross) if cross in newAttributes else None)
             for cross in crossDelete
